src/main/java/spoon/support/compiler/jdt/JDTBasedSpoonCompiler.java

```diff
@@ -430,8 +430,11 @@ protected boolean buildUnitsAndModel(JDTBuilder jdtBuilder, SpoonFolder sourcesF
 
 	protected void buildModel(CompilationUnitDeclaration[] units) {
 		JDTTreeBuilder builder = new JDTTreeBuilder(factory);
+		List<CompilationUnitDeclaration> unitList = Arrays.asList(units);
+		unitList.sort(new CompilationUnitComparator());
```

Comment:
Is it intentional that this sort sorts the origin `units` array? It actually modifies the array sent from outside. I have checked who actually calls it and it looks like it is actually no problem, but it would be safer to sort a copy of that array.
WDYT?

Reply:
You're right: I wasn't aware that `Arrays.asList` was keeping up the link with the original array. Good thing to know 👍